src/pylexibank/cldf.py

```diff
@@ -248,6 +248,13 @@ def add_cognate(self, lexeme=None, **kw):
 
     def add_language(self, **kw):
         if 'Glottocode' in kw \
+                and 'Latitude' in kw \
+                and 'Longitude' in kw \
+                and 'Macroarea' in kw \
+                and 'Family' in kw \
+                and 'ISO6339P3code' in kw:
```

Comment:
Hm. That's a bit of a hack, though, because
- it doesn't check for `Glottolog_name`
- it will prevent looking up **authoritative** info (`Macroarea`, `Family` and `Glottolog_name`) from the appropriate Glottolog version.

So I'm somewhat reluctant to merge. I guess a proper command line switch for `lexibank.makecldf` like `--quick-and-dirty` or similar  would be a better solution.

Reply:
Maybe that's also something we should add to the lexibank HowTo guide: While it's ok to provide more appropriate geo-coords for languoids, macroarea, family and glottolog name should **not** be provided!

Reply:
Okay, then a compromise: can we add a command line argument that allows me to NOT call the glottolog api for all langoids? This *is* making it very hard to work quickly on certain datasets, no matter what computer I am using.

Reply:
So something like a development mode.

Reply:
`cldfbench lexibank.makecldf sagartst --dev`

Reply:
Yes, sounds like a good option name.

Reply:
Thanks for this fast solution!